core/capabilities/webapi/target/target.go

```diff
@@ -57,6 +57,10 @@ func (c *Capability) Start(ctx context.Context) error {
 }
 
 func (c *Capability) Close() error {
+	err := c.registry.Remove(context.TODO(), c.capabilityInfo.ID)
```

Comment:
For `Close()` methods, we typically use a sanity check like `1s` so that a service cannot block shutdown or whatever else:
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
	defer cancel()
	err := c.registry.Remove(ctx, c.capabilityInfo.ID)
```

Reply:
Ah, good tip. Added that!